docs/faq.rst

```diff
@@ -399,7 +399,7 @@ Create a wrapper script:  /usr/local/bin/pv-wrapper  ::
 
 Add BORG_RSH environment variable to use pipeviewer wrapper script with ssh. ::
 
-    export BORG_RSH='/usr/local/bin/pv-wrapper.sh ssh'
+    export BORG_RSH='/usr/local/bin/pv-wrapper ssh'
```

Comment:
thanks, good catch!

could you do a PR against 1.0-maint, so it gets into next 1.0.x release? it will automatically be merged into master from there also.